rust/Cargo.lock

```diff
@@ -37,6 +37,11 @@ version = "0.8.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
Yes, my phrasing was maybe a bit misleading. I meant more like, "can we _please_ exclude the Cargo.lock from this PR?" ;)